docker-for-windows/release-notes.md

```diff
@@ -16,10 +16,21 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop 3.2.2
+2021-03-15
+
+> [Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe)
+
+### Bug fixes and minor changes
+
+- Allow exposing port 53 (DNS) without clashing with an internal DNS server. Fixes [docker/for-win#10601](https://github.com/docker/for-win/issues/10601).
+- Fixed an issue running 32bit Intel binaries in emulation on Intel CPU's. Fixes [docker/for-win#10594](https://github.com/docker/for-win/issues/10594).
+- Fixed an issue related to high CPU consumption and frozen UI when network connection lost. Fixes [for-win/#10583](https://github.com/docker/for-win/issues/10563).
```

Comment:
```suggestion
- Fixed an issue related to high CPU consumption and frozen UI when network connection is lost. Fixes [for-win/#10583](https://github.com/docker/for-win/issues/10563).
```

Reply:
Is there a typo in the GitHub issue number? Should it be 10563?

Reply:
👀 Oh yes, that was a typo. Thank you!

Reply:
Yes, 63 (as in the link) not 83 (as in the text).